lib/index.ts

```diff
@@ -146,6 +149,35 @@ async function dispatchCommand(
         }
     }
 }
+const handleFloorPriceChannel = async (client: any, magicEdenData: any)  => {
+    try {
+        const myChannel = client.channels.cache.get(config.priceChannel);
+        const price = Number(magicEdenData.floorPrice)/LAMPORTS_PER_SOL
+        myChannel.setName(`Floor Price: ◎${price}`);
+    } catch(error) {
+        console.log(error);
+    }
+
+};
+
+const handleTotalVolumeChannel = async (client: any, magicEdenData: any)  => {
+    try {
+        const myChannel = client.channels.cache.get(config.volumeChannel);
+        const volume = numberWithCommas((Math.round(magicEdenData.volumeAll/LAMPORTS_PER_SOL)).toString())
+        myChannel.setName(`Total Volume: ◎${volume}`);
+    } catch(error) {
+        console.log(error);
+    }
+};
+
+function channelNamesRunner(client: Client, magicEdenData: object) {
+    handleFloorPriceChannel(client, magicEdenData);
+    handleTotalVolumeChannel(client, magicEdenData);
+    setTimeout(function() {
+        channelNamesRunner(client, magicEdenData);
```

Comment:
This doesn't work you are always using the old ME data you need a function which fetches it and passes it not just fetch once then pass